src/ragas/testset/synthesizers/generate.py

```diff
@@ -267,6 +269,42 @@ def generate_with_llamaindex_docs(
             raise_exceptions=raise_exceptions,
         )
 
+    def generate_persona_list(
+        self,
+        generation_group: Callbacks,
+        run_config: t.Optional[RunConfig] = None,
+        raise_exceptions: bool = True,
+    ) -> PersonaList:
```

Comment:
just noting it here that we can another implementation of this in the PR before this - still up for review